VirtualSatelliteCAD/Resources/Ui/.gitignore

```diff
@@ -0,0 +1 @@
+/preferences.ui
```

Comment:
Is the preference.ui file intended to be in the .gitignore?

Reply:
Thanks for the review! 
Actually this is intentionally: This file is build at runtime from the other *.ui files in the folder to also contain entries for each plugin (which we only know at runtime). It is then saved as a complete file because the FreeCAD interface expects a file path for a settings menu. Do you have any suggestions how to make this more clear?

Reply:
Ok, understood.